public/src/components/channelManagement/epicTests/tagsEditor.tsx

```diff
@@ -100,11 +100,11 @@ export const TagsEditor: React.FC<TagEditorProps> = ({
       renderInput={(params): JSX.Element => (
         <TextField {...params} variant="outlined" label={label} />
       )}
-      renderOption={(option): JSX.Element => {
+      renderOption={(props, option): JSX.Element => {
         return <div>{option.name ? `${option.name} (${option.id})` : option.id}</div>;
       }}
       onChange={(event, values: Tag[], reason): void => {
-        if (reason === 'select-option' || reason === 'remove-option') {
+        if (reason === 'selectOption' || reason === 'removeOption') {
```

Comment:
what changed here?

Reply:
The values provided by MUI changed slightly. Luckily the typescript caught it because the types are well defined!
See https://mui.com/material-ui/api/autocomplete/#autocomplete-prop-onChange